src/yardstick/store/label_stats.py

```diff
@@ -0,0 +1,73 @@
+from __future__ import annotations
+
+import hashlib
+import json
+import logging
+import os
+import shutil
+from typing import Any, List
+
+from yardstick import comparison
+from yardstick.store import config as store_config
+
+LABEL_STATS_DIR = "label-stats"
+
+
+def _store_root(store_root: str | None = None) -> str:
+    if not store_root:
+        store_root = store_config.get().store_root
+    return os.path.join(store_root, LABEL_STATS_DIR)
+
+
+def store_path(ids: List[str], configuration: list[dict[str, Any]] | None, store_root: str | None = None) -> str:
+    config_str = _derive_configuration_string(configuration)
+
+    filename = "_".join(sorted(ids)) + "_" + config_str + ".json"
+
+    return os.path.join(_store_root(store_root=store_root), filename)
+
+
+def clear(store_root: str | None = None):
+    try:
+        shutil.rmtree(_store_root(store_root=store_root))
+    except FileNotFoundError:
+        pass
+
+
+def save(result: comparison.ImageToolLabelStats, store_root: str | None = None):
+    if not isinstance(result, comparison.ImageToolLabelStats):
+        raise RuntimeError(f"only ImageToolLabelStats is supported, given {type(result)}")
+
+    ids = [c.ID for c in result.configs]
+    path = store_path(ids, result.compare_configs, store_root=store_root)
+    logging.debug(f"storing label comparison state for {ids!r}")
+
+    os.makedirs(os.path.dirname(path), exist_ok=True)
+
+    with open(path, "w", encoding="utf-8") as data_file:
+        data_file.write(json.dumps(result.to_dict(), indent=2))
+
+
+def _derive_configuration_string(configurations: list[dict[str, str]] | None) -> str:
```

Comment:
This is a configuration digest/sha/fingerprint, right? `configuration_string` makes me think it's a representation of the config.